### DISCUSSION TITLE
Not Open

### DISCUSSION THREAD

**Discussion by rabendrashah** [community/community#120214] [Mobile] [answered]
### Select Topic Area

Question

### Body

Not opne Meta Mask my Android phone

**Comment by stevepopovich** [comment]
Hi @rabendrashah do you have a specific bug or issue with the iOS or Android app?

**Reply by rabendrashah** [reply]
Yes

**Reply by rabendrashah** [reply]
I have Android

**Comment by StackSorcerer403** [comment, marked as answer]
1️⃣ Restart MetaMask: Close the app completely and reopen it.
2️⃣ Restart Your Phone: Turn off your phone and then turn it back on.
3️⃣ Update MetaMask: Check for updates in the Google Play Store.
4️⃣ Clear Cache: Go to "Settings" > "Apps" > "MetaMask" > "Storage" > "Clear Cache."
5️⃣ Reinstall MetaMask: Uninstall and reinstall the app from the Google Play Store.
6️⃣ Check for Android Updates: Ensure your phone's operating system is up to date.
7️⃣ Contact MetaMask Support: If the problem continues, reach out to MetaMask's support team

**Reply by rabendrashah** [reply]
Not start in my phone MetaMask
